src/data/real/ennreal.lean

```diff
@@ -1232,6 +1232,19 @@ begin
   rw mul_div_cancel' h0 hI, exact le_refl b
 end
 
+lemma eq_div_iff (ha : a ≠ 0) (ha' : a ≠ ∞) :
+  b = c / a ↔ a * b = c :=
```

Comment:
TODO (possibly, for the future): add this lemma for an `is_unit` in any `division_monoid`.